tests/phpunit/tests/functions/_wpCheckExistingFileNames.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * @group functions.php
+ *
+ * @covers ::_wp_check_existing_file_names
+ */
+class Tests_Functions__wp_check_existing_file_names extends WP_UnitTestCase {
+
+	/**
+	 * @ticket 55192
+	 *
+	 * @dataProvider data_wp_check_existing_file_names
+	 *
+	 * @param string $filename filename looked for.
+	 * @param array $files files check against.
+	 * @param boolean $expected Expected result.
```

Comment:
While this file seems part of another ticket, a suggestion for that ticket:

```suggestion
	 * @param string $filename The file name to check.
	 * @param array  $files    An array of existing files in the directory.
	 * @param bool   $expected The expected result.
```
- Alignment, capitalization, param descriptions updated to match the descriptions for `_wp_check_existing_file_names()`
- Core uses `bool`, not `boolean` for `@param` tags.